runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8591,10 +8591,10 @@ class INTERPRETER_CLASS
 			}
 		}
 
-		/* In the future Valhalla checkcast needs to throw exception on
-		 * null restricted checkedType if obj is null,
-		 * see issue https://github.com/eclipse-openj9/openj9/issues/19764
-		 */
+		/* In the future, Valhalla checkcast must throw an exception on
+		* null-restricted checkedType if object is null.
+		* See issue https://github.com/eclipse-openj9/openj9/issues/19764.
+		*/
```

Comment:
Please correct the indentation here (the `*`s should line up as before).

Reply:
Fixed.